.prettierrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "singleQuote": true,
+  "jsxSingleQuote": true,
```

Comment:
I think we should keep `jsxSingleQuote` to be double-quoted as it is best practice.